tests/unit/disco/components/TestAddon.js

```diff
@@ -378,71 +376,6 @@ describe(__filename, () => {
     });
   });
 
-
-  describe('Theme Previews', () => {
-    let root;
-    let themeImage;
-    let previewTheme;
-    let resetThemePreview;
-
-    beforeEach(() => {
-      previewTheme = sinon.spy();
-      resetThemePreview = sinon.spy();
-      const data = {
-        ...result,
-        type: ADDON_TYPE_THEME,
-        previewTheme,
-        resetThemePreview,
-      };
-      root = renderAddon({ addon: data, ...data });
-      themeImage = root.find('.theme-image');
-    });
-
-    it('runs theme preview onHoverIntent on theme image', () => {
-      const onHoverIntent = root.find(HoverIntent).prop('onHoverIntent');
-      onHoverIntent({ currentTarget: fakeEvent.currentTarget });
-      sinon.assert.calledWith(previewTheme, fakeEvent.currentTarget);
-    });
-
-    it('resets theme preview onHoverIntentEnd on theme image', () => {
-      const onHoverIntentEnd = root.find(HoverIntent).prop('onHoverIntentEnd');
-      onHoverIntentEnd({ currentTarget: fakeEvent.currentTarget });
-      sinon.assert.calledWith(resetThemePreview, fakeEvent.currentTarget);
-    });
-
-    it('runs theme preview onFocus on theme image', () => {
-      themeImage.simulate('focus', fakeEvent);
-      sinon.assert.calledWith(previewTheme, fakeEvent.currentTarget);
-    });
-
-    it('resets theme preview onBlur on theme image', () => {
-      themeImage.simulate('blur', fakeEvent);
-      sinon.assert.calledWith(resetThemePreview, fakeEvent.currentTarget);
-    });
-
-    it('calls installTheme on click', () => {
```

Comment:
As mentioned above, do we not still want this to work, and should we therefore keep this test?

Reply:
as far as i can see, clicking on anything but the switch does not install the theme so i don't * think * we need this (more info on comments above about this)

Reply:
What's odd about that is that this test was evidently passing before you removed any code, but it sounds like it should have been failing. @muffinresearch Can you comment on this? Are you supposed to be able to install a theme by clicking on the image from disco pane?

Reply:
> What's odd about that is that this test was evidently passing before you removed any code, but it sounds like it should have been failing. @muffinresearch Can you comment on this? Are you supposed to be able to install a theme by clicking on the image from disco pane?

Yep as mentioned in previous comments above, clicking the image also carries out an install.

Reply:
Hmm having just checked it, click installs don't appear to currently work.